applications/crossbar/doc/api_authentication.md

```diff
@@ -32,9 +32,6 @@ Key | Description | Type | Default | Required
 --- | ----------- | ---- | ------- | --------
 `api_key` | The Accounts API key | `string(64)` |   | `true`
 
-
-
-
```

Comment:
these were here for scripts/generate-doc-schemas.sh
I don't remember the exact amount of newlines needed so not sure if this breaks that. Can you check?